src/test/java/com/google/api/codegen/GapicCodeGeneratorAnnotationsTest.java

```diff
@@ -36,20 +36,20 @@ public GapicCodeGeneratorAnnotationsTest(
       String packageConfigFileName,
       List<String> snippetName,
       String apiName,
-      String baseline) {
-    super(language, null, null, snippetName, baseline);
+      String baseline,
+      String protoPackage) {
+    //    super(language, null, null, snippetName, baseline);
+    super(language, null, null, snippetName, baseline, "google.example.library.v1");
```

Comment:
Should "google.example.library.v1" instead be `protoPackage`? Otherwise protoPackage is unused.

Reply:
Yes, thanks :)